circle.yml

```diff
@@ -82,10 +78,57 @@ jobs:
             make install_system_dependencies
             sudo ln -s /usr/lib/x86_64-linux-gnu/libmysqlclient.so /usr/lib/x86_64-linux-gnu/libmysqlclient.so.18
             bin/install-supported-firefox
-            bin/run-circleci-tests
-          no_output_timeout: 6h
+            mkdir -p /tmp/coverage
+            make << parameters.tests_type >>
+            if [ -e .coverage.* ]; then
+                cp -v .coverage.* /tmp/coverage/.
+            fi
+            if [ -d coverage ]; then
+                cp -vR coverage/* /tmp/coverage/.
+            fi
+          no_output_timeout: 2h
           environment:
             ANSIBLE_HOST_KEY_CHECKING: False
+      - persist_to_workspace:
+          root: /tmp
+          paths:
+              - coverage
+  coverage:
+    docker:
+      - image: circleci/python:3.5.4-node
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependencies-{{ checksum "requirements.txt" }}
+      - run:
+          name: Install Dependencies
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install --upgrade pip
+            pip install --upgrade virtualenv
+            # The jessie-updates and jessie-backports areas are now removed from the repositories. So
+            # the lines corresponding to those areas have to be removed to fix the resulting errors.
+            # This can be removed after upgrading to the image based on Debian Stretch.
+            sudo sed -i '/jessie-updates/d' /etc/apt/sources.list
+            sudo sed -i '/jessie-backports/d' /etc/apt/sources.list
```

Comment:
Is this still relevant after changing the image to a `stretch` one?

Reply:
@viadanna I don't think so. I'm rebasing with master and taking care of this. Thanks.